cmd/bootstrap/runner.go

```diff
@@ -289,14 +296,119 @@ func (r *runner) installAppCatalogs(ctx context.Context, k8sClients k8sclient.In
 	return nil
 }
 
+func (r *runner) ensureChartMuseumPSP(ctx context.Context, k8sClients k8sclient.Interface) error {
+	name := "chartmuseum-psp"
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("ensuring psp %#q", name))
+
+	t := true
```

Comment:
What's the canonical way of specifying `true` to be used in Line 360?

Reply:
You can use our `to` library for this. https://github.com/giantswarm/to/blob/ac0821f81a8e530c3e12f0dca38312aafe5160ac/to.go#L41

```go
AllowPrivilegeEscalation: to.BoolP(true),
```

Reply:
:) thanks